docs/changelog_for_devs.md

```diff
@@ -21,6 +21,31 @@ APIs/RPC interface.
 
 - ⚠️ Add `outsider` field to `MarketBonds` struct. In particular, the `Market`
   struct's layout has changed ([#903]).
+  # v0.3.9
+
+All things about Global Disputes:
+- Added new dispatchable function:
+  - `refund_vote_fees` - Return all vote funds and fees, when a global dispute
+    was destroyed.
+- Added the following events:
+  - `OutcomeOwnerRewarded` for `Possession::Paid`
+  - `OutcomeOwnersRewarded` for `Possession::Shared`
+  - `OutcomesFullyCleaned` and `OutcomesPartiallyCleaned` for extrinsic `refund_vote_fees`
+- Removed the following events:
+- Added enum `Possession` with variants:
+  - `Paid { owner: AccountId, fee: Balance }`
+  - `Shared { owners: BoundedVec }`
+- `OutcomeInfo` has the following fields:
+  - `outcome_sum: Balance`
+  - `possession: Possession`
+- Replace `WinnerInfo` by `GlobalDisputeInfo` with the following fields:
+  - `winner_outcome: OutcomeReport`
```

Comment:
It seems like some of these changes are breaking and should be marked accordingly.

Reply:
Since the global dispute pallet was only used on battery station, I think it's fine. What would you like me to write down?